ev3dev2/motor.py

```diff
@@ -1725,10 +1822,8 @@ def __init__(self, left_motor_port, right_motor_port, desc=None, motor_class=Lar
         self.max_speed = self.left_motor.max_speed
 
     def _block(self):
```

Comment:
Thoughts on moving this up to the MotorSet class?

Reply:
Makes sense. Same applies to the `off()` method below.

Reply:
Filed #540 for the off() part